teamMaker2.py

```diff
@@ -1,66 +1,126 @@
 import itertools
+import pickle
+import os.path
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly']
+
+SPREADSHEET_ID = '1c2Kvc3y1GrF9EFqAKvPoRU8NmFV_exCrAeCTrmDct2Y'
+ELOS_RANGE = 'TEST!A2:ZZ12'
```

Comment:
lol, someone better not delete a row.

I have some seen some crappy DBs, google sheets now takes the crown

Reply:
We could add markings to the sheet to indicate where one section starts and the next ends, or maybe put different sections of the sheet in different tabs.